src/main/java/frc/robot/Robot.java

```diff
@@ -57,7 +77,8 @@ public void teleopExit() {
 
     @Override
     public void autonomousInit() {
-
+        //swerve.resetEncoders();
+        autoChooser.getSelected().schedule();
```

Comment:
also check that a command was selected. In reality `null` may be returned.

Additionally, just to be safe, cancel this command in `autonomousExit`